src/label_extraction.py

```diff
@@ -201,11 +201,26 @@
 # %% [markdown]
 # ### Analyze the population
 
-# The age bias in OASIS is well known and this is why the youngest CN
-# participants were previously excluded. However, other biases may exist,
-# especially after the quality check of the preprocessing which removed sessions
-# from the dataset. Thus, it is crucial to check before going further if there
-# are other biases in the dataset.
+# You may have noticed that there is a bias on age in the OASIS dataset: the youngest AD
+# patient is 62 years old, whereas many CN patients are much younger. To correct this bias,
+# we will remove the youngest CN patients (i.e. who are less than 62 years old) from our data, 
+# thanks to the following Python script:
+
+# %%
+def remove_youngest_cn(table_path, minimum_age):
+    """Remove younger CN patients to correct age bias"""
+    import pandas as pd
+
+    table = pd.read_csv(table_path, sep='\t')
+    new_table = table[(table['diagnosis'] == 'AD') | (table['age_bl'] >= minimum_age)]
```

Comment:
It would be nice to have some info on what was removed.
For example by printing the list of subject IDs that were discarded.
WDYT?